source/common/utility.cc

```diff
@@ -72,4 +72,17 @@ void Utility::parseCommand(TCLAP::CmdLine& cmd, const int argc, const char* cons
   }
 }
 
-} // namespace Nighthawk
\ No newline at end of file
+HostAddressType Utility::hostAddressTypeFromHostPort(const std::string& host_port) {
```

Comment:
Note: This function no longer classifies the address, it just splits out the :port. I still kept this code separate because it's very short. I started relying on the resolver, with its existing unit tests, to catch malformed addresses later.